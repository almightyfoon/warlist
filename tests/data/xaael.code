cf101b_-12l8lslulvlFfC9ulGlElElElmlAlklkiBlD___-12l9lclF7jlH7kfClGlxlylelgljlkld
