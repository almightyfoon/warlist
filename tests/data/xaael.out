https://conflictchamber.com/#cf101b_-12l8lslulvlFfC9ulGlElElElmlAlklkiBlD___-12l9lclF7jlH7kfClGlxlylelgljlkld

Crucible Guard Army - 75 / 75 points
[Theme] Magnum Opus

[Syvestro 1] Aurum Adeptus Syvestro [+28]
 - Retaliator [9]
 - Toro [13]
 - Vindicator [15]
 - Aurum Ominus Alyce Marc [0(5)]
Hutchuk, Ogrun Bounty Hunter [0(6)]
Ragman [4]
Rocketman Ace [4]
Trancer [3]
Trancer [3]
Trancer [3]
Crucible Guard Assault Troopers (max) [13]
Crucible Guard Storm Troopers (max) [16]
Dragon's Breath Rocket [0(5)]
Dragon's Breath Rocket [5]
Steelhead Halberdiers (max) [11]
 - Doctor Alejandro Mosby [4]

Crucible Guard Army - 75 / 75 points
[Theme] Magnum Opus

[Gearhart 1] Marshal General Baldwin Gearhart [+27]
 - Vulcan [35]
 - Aurum Ominus Alyce Marc [0(5)]
Anastasia di Bray [3]
Crucible Guard Mechanik [2]
Gorman di Wulfe, Rogue Alchemist [4]
Hutchuk, Ogrun Bounty Hunter [0(6)]
Rocketman Ace [4]
Crucible Guard Infantry (max) [15]
 - Crucible Guard Infantry Officer & Standard [4]
Crucible Guard Rocketmen (min) [9]
 - Crucible Guard Rocketman Captain [4]
 - Rocketman Gunner (3) [6]
Dragon's Breath Rocket [0(5)]
Railless Interceptor [16]
